src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -68,12 +68,9 @@ void processFeatureForOutput(OpenMS::Feature& curr_feature, bool write_convex_hu
   // Ensure a unique id is present
   curr_feature.ensureUniqueId();
 
-  // Sum up intensities of the MS2 features
-  if (curr_feature.getMZ() > quantification_cutoff_ && ms_level == "MS2")
-  {
-    total_intensity += curr_feature.getIntensity();
-    total_peak_apices += (double)curr_feature.getMetaValue("peak_apex_int");
-  }
+  // Sum up intensities of the features
+  total_intensity += curr_feature.getIntensity();
+  total_peak_apices += (double)curr_feature.getMetaValue("peak_apex_int");
```

Comment:
maybe we should split MS1 and MS2 intensity - does this here sum up the total intensity across MS1 and MS2 features? is that a good idea?

Reply:
MS1 and MS2 signals are summed up separately, combining them would probably be a bad idea. The function is called twice and ``total_intensity`` and ``total_peak_apices`` are attached as separate MS1 and MS2 meta values to the main feature for easy access when writing the reports.